proposed/http-message.md

```diff
@@ -112,7 +112,7 @@ request or response.
 
 In order to accommodate headers with multiple values yet still provide the
 convenience of working with headers as strings, headers can be retrieved from
-an instance of a ``MessageInterface`` as an array or string. Use the
+an instance of a ``MessageInterface`` as an array or a string. Use the
```

Comment:
For some reason, there are duplicated code tags around the word MessageInterface; could you fix those here? (They were in the original, too.)